include/postprocessors/ElementVectorL2Difference.h

```diff
@@ -1,24 +0,0 @@
-#include "ElementIntegralPostprocessor.h"
-
-/**
- * Computes the L2-Norm difference between two solution vector fields.
- */
-class ElementVectorL2Difference : public ElementIntegralPostprocessor
```

Comment:
Does Moose's ElementL2Difference replace this functionality or is this removing some different functionality that we just don't need?

Reply:
Removing some different functionality - MOOSE's `ElementL2Difference` doesn't handle vector fields. We still have this in Apollo (as it's used there in transfer tests) in case we end up upstreaming it to MOOSE, but there's no reason for it to need to live in Platypus from what I can tell

Reply:
Ok, thanks